### PR TITLE
Increased usability in the UI and created a seed data script

### PR DESCRIPTION
**Pull request by dmsakamoto**
Added whether or not the user is currently in a sheep/shepherd in the nav bar and created a seed data script

### CODE REVIEW

**Comment by dmsakamoto** [comment]
This fixes #80 and #125

**Comment by kevhuang** [comment]
For the seed data, it may be better to also associate some of the users as sheep to an event, or as a shepherd to an event.

For the UI change, it's a bit confusing with the tagline and button. They seem to contradict one another. For example, it'll say, "Participate in events" and then the button displays the word, "Shepherd."

Maybe we can move the button to the right side and have some sort of label next to it saying something like, "Toggle to:"? I'm open to ideas.

**Comment by dmsakamoto** [comment]
For the UI, the current set up is confusing. That's my point. We can change it to whatever, but just do it if you have a better idea

**Comment by kevhuang** [comment]
:+1:  
We can touch up the UI later.

File: client/public/js/components/App.js [hunk 1]
```diff
@@ -122,6 +122,7 @@ var App = React.createClass({
                       It will be disabled when viewing the event-create and event-detail components
                     */}
                     <li>
+                      <span>{this.state.mode === 'shepherd' ? 'Manage your events' : 'Participate in events'}</span>
```
Comment (ekong2) [inline client/public/js/components/App.js:125]: I think you meant to reverse these two? 

this.state.mode === 'shepherd' ? 'Participate in events' : 'Manage your events'

File: server/config/seed.js [hunk 2]
```diff
@@ -1 +1,68 @@
...
+db.User.create({
+  username: 'Johnny Tsunami'
+}).then(function (user) {
+  Event.create({
+    name: 'I see you....',
+    location: 'Lecture Area',
+    start: new Date(),
+    end: new Date(currentDate.getTime() + 60 * 60 * 24 * 1000)
+  }).then(function (event) {
+    Observation.create({
```
Comment (ekong2) [inline server/config/seed.js:35]: will have to add in a rawImage column later

**Comment by kevhuang** [comment]
As discussed, @dmsakamoto please submit a new PR with only the UI changes. @stevenshuhyow will work on the seed data script.

**Pull request closed without merging**